hadoop-hdds/docs/content/security/SecuringDatanodes.zh.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "安全化 Datanode"
+date: "2019-April-03"
+weight: 2
+summary:  解释安全化 datanode 的不同模式，包括 Kerberos、证书的手动颁发和自动颁发等。
```

Comment:
Question about original doc, which is more precise, `datanode` or `data node` ?

Reply:
datanode